audio/out/ao_pulse.c

```diff
@@ -294,6 +294,12 @@ static int pa_init_boilerplate(struct ao *ao)
     struct priv *priv = ao->priv;
     char *host = priv->cfg_host && priv->cfg_host[0] ? priv->cfg_host : NULL;
     bool locked = false;
+    pa_proplist *props = pa_proplist_new();
```

Comment:
Can this fail and return NULL? If so we need to catch it, or at least use some sort of MP_HANDLE_OOM